cmd/go-clang-generate/main.go

```diff
@@ -201,7 +201,7 @@ func main() {
 	clangIndexHeaderFilepath := "./clang-c/Index.h"
 	tu := idx.Parse(clangIndexHeaderFilepath, []string{
 		"-I", ".", // Include current folder
-		"-I", "/usr/local/lib/clang/3.4.2/include/", // Include clang headers TODO make this generic
+		"-I", "/usr/include/clang/3.6.2/include/", // Include clang headers TODO make this generic
```

Comment:
Please add your include folder additionally to the existing include folder. I will figure out later how to resolve this, but removing mine breaks the generation on my side.